src/components/TranslationDialog/TranslationModal/TranslationForm.js

```diff
@@ -73,17 +73,16 @@ export const TranslationForm = ({
         )
     }
 
-    const [fetchFieldsTranslations] = useDataMutation(
-        {
-            resource: 'i18n',
-            type: 'create',
-            data: fieldsToTranslate.map(camelCaseToUnderscores),
-        },
-        {
-            onComplete: (res) => setFieldsTranslations(res),
-            onError: (error) => showError(error),
-        }
-    )
+    const i18nMutationRef = useRef({
```

Comment:
Is it possible to put these mutation objects (i18nMutation and translationsMutation on line 87) outside of the component rather than using useRef? I'm not against useRef at all, just wondering.  Also, looks like `camelcaseToUnderscores` does not depend on the component state, so it could (and probably should) be moved outside of the component.